freqtrade/optimize/__init__.py

```diff
@@ -97,3 +97,19 @@ def start_hyperopt(args: Namespace) -> None:
         # TODO: return False here in order to help freqtrade to exit
         # with non-zero exit code...
         # Same in Edge and Backtesting start() functions.
+
+
+def start_edgecli(args: Namespace) -> None:
```

Comment:
why not start_edge()?
(and RunMode.EDGE i.o. RunMode.EDGECLI?)

Reply:
well - the class (and file) are called EdgeCLI - to avoid confusion with Edge (in another namespace/package) - so that makes sense - and honestly, i just named it according to the class (but yes, using edge would make sense here).

Reply:
I understand that don't like edge_cli in common. Or there should be hyperopt_cli and backtesting_cli  as well. But this is not for this PR...

LGTM 👍